### DISCUSSION TITLE
Does delay + transform really not work on Safari native WAAPI?

### DISCUSSION THREAD

**Discussion by tinleym** [motiondivision/motionone#166] [Support] [answered]
This isn't a Motion One issue... I'm just hoping someone here might know something about this.

The following call works on Chrome, but doesn't work on Safari. It works on Safari after delay is set to 0 or removed.

```
element.animate(
  { transform: "translateY(-100vh)" },
  {
    delay: 1,
    duration: 250,
  }
)
```

This seems like a pretty conspicuous bug, but I can't find any other reference to it anywhere... which makes me think Safari might be more strict and that I might be invoking the delay incorrectly, but I don't see how. Anyway, if anybody can say if this is a known issue, or that it works for you, I'd love to know!

**Comment by mattgperry** [comment]
It’d be interesting to know if it is still broken if you use opacity instead, or if you change vh to px what that does (non px units need to be resolved into px before the browser can apply them)

**Comment by tinleym** [comment, marked as answer]
Ah... it turns out some polyfill was eating resources and this is where it first popped up. All better now!
